package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kano/api-client",
-  "version": "3.4.9",
+  "version": "3.4.10",
```

Comment:
this will be updated by the `yarn publish` when you go to publish it

Reply:
yeh it updated because I did publish it...

Reply:
Ah, surely it should be merged to master before publishing though?

Reply:
Yes probably, I've been merging my own PRs with the client with Pv and Radek just checking in now and then to make sure things are ok. This was more of a "look heres whats changing", rather than "can you review this code" - its only one api change...